database.rules.json

```diff
@@ -135,6 +135,10 @@
         "
       }
     },
+    "destinations":{
```

Comment:
@scboesch please review the database rules

Reply:
These rules don't allow removing destinations if I get it right. Do we need to remove them? Also, is there a typo in `orginanator`  field?

Reply:
@scboesch kindly suggest.

Reply:
Here are the firebase rules for destinations
```
 "destinations":{
      ".read" : "auth != null",
      "$destinationKey" :{
        ".read" : "auth != null",
        ".write" : "
            auth != null &&  
            (
              ( 
                newData.exists() && 
                newData.child('originator').exists() &&
                newData.child('originator').val() == auth.uid && 
                ( 
                  !data.exists()  || 
                  ( data.child('originator').val() == auth.uid) 
                ) 
              ) || ( 
                data.exists() && data.child('originator').val() == auth.uid 
              )
            )"
        }
    },
```